### ISSUE TITLE
Can't add "now playing" item in customization on 0.6

### ISSUE THREAD

**Issue by semihcanturk** [pigigaldi/Pock#207]
**Describe the bug**
When trying to customize touch bar items, the "now playing" item is unavailable in the customization screen

**How to reproduce**
```
1. Menu bar -> Pock -> Customize (cmd + C)
```

**Expected behavior**
A draggable "now playing" item icon should appear.

**Screenshots**

<img width="1279" alt="Screen Shot 2019-08-21 at 2 47 50 PM" src="https://user-images.githubusercontent.com/20279851/63429718-7e195680-c423-11e9-8906-0da5b9625b28.png">
<img width="1085" alt="Touch Bar Shot 2019-08-21 at 2 52 42 PM" src="https://user-images.githubusercontent.com/20279851/63429730-85406480-c423-11e9-86b8-245a4b0e9bf1.png">

**Versions:**
 - macOS version: 10.14.6
 - Pock version:  0.6

**Additional context**
Not entirely sure if related, but the original control strip on the touch bar persists in the Customize screen (see screenshot above) even when "Hide Control Strip" option is checked, potentially limiting the number of items one may add to the touch bar during customization.  Once the customization is done, the control strip does disappear as it should. This may warrant a separate issue.

**Comment by isofia** [comment]
same as me

**Comment by Fullchee** [comment]
I had the same thing.

I uninstalled pock and reinstalled it without success. 

I think restarting my computer got it back for me.

**Comment by martinschilliger** [comment]
I had the issue as well. Turns out, it doesn't detect spotify if it is playing through another speaker, not the mac itself.

**Comment by pigigaldi** [comment]
@semihcanturk This happens because, if nothing is currently playing, `Now Playing` widget is hidden by default. Maybe I have to make it visible in the customization window even if nothing is playing.
I also need to find a "private-way" to hide the "original control strip" that persists during customization.

@martinschilliger this is explained here: https://github.com/pigigaldi/Pock/issues/196
